### ISSUE TITLE
WarmFuzzies: update how we use the brain...

### ISSUE THREAD

**Issue by pcunning** [rogue-hack-lab/hackbot#16]
instead of storing each value under a different key we should store all values for wf under the same key. Something like:
```
robot.brain.set(warm_fuzzy, {
    pcunning: 10,
    kconnor: 20
});
```
or in coffee script
```
robot.brain.set warm_fuzzy,
  pcunning: 10
  kconnor: 20
```
This should also eliminate the need to store the "keys"

**Comment by pcunning** [comment]
closed with #23

**Comment by pcunning** [comment]
closed with #23
